addons/account/static/tests/tours/tour_tests_search_read.js

```diff
@@ -0,0 +1,32 @@
+import { accountTourSteps } from "@account/js/tours/account";
+import { registry } from "@web/core/registry";
+
+registry.category("web_tour.tours").add('search_read_tour', {
+    url: "/odoo",
+    steps: () => [
+    ...accountTourSteps.goToAccountMenu("Go to Invoicing"),
+    {
+        content: "Go to Customers",
+        trigger: 'span:contains("Customers")',
+        run: "click",
+    },
+    {
+        content: "Go to Invoices",
+        trigger: 'a:contains("Invoices")',
+        run: "click",
+    },
+    {
+        content: "Create new invoice",
+        trigger: '.o_control_panel_main_buttons .o_list_button_add',
+        run: "click",
+    },
+    {
+        content: "Click Catalog Button",
+        trigger: 'button[name=action_add_from_catalog]',
+        run: "click",
+    },
+    {
+        content: "Load Product Category Page",
+        trigger: 'b:contains("Product Category")',
+    },
```

Comment:
I think it's worth going a bit further: select a product and make sure there is a now AML for the product